BDT_training/step2_BDTtraing_condor/prepare_condor.py

```diff
@@ -27,7 +32,7 @@ def GetBackgroundList(era):
   jsonfile.close()
   BackgroundFile_List = []
   for process, desc in samples:
-    if desc[4] == 1:
+    if desc[4] == 1: # Correct from desc[5] to desc[4] to be consistent with other steps
```

Comment:
We know this comments from last PR , #33 
so I think better to remove ``# Correct from desc[5] to desc[4] to be consistent with other steps``